lab1.js

```diff
@@ -78,16 +83,23 @@ var sentence2 = 'Come over here so you can scratch my belly.';
 // 'chirp' (10 points)
 
 // your code goes here
-
+for (var i = 0; i < sentence1Split.length; i++) {
+  sentence1Split[i] = sentence1Split[i].replace('More', 'chirp').replace('food', 'chirp').replace('please', 'chirp');
+  sentence1 = sentence1Split.join(' ');
+}
 // TODO: part #2: use a while or do-while loop to replace the words in sentence 2
 // with 'chirp' (10 points)
 
 // your code goes here
+while (i <= sentence2Split.length) {
+  sentence2Split[i] = sentence2Split[i].replace('Come', 'chirp').replace('over', 'chirp').replace('here', 'chirp')
```

Comment:
this totally works. And, I'll bet you can figure out a way to do this that doesn't involve hard-coding in the values you're replacing.